import_manager.go

```diff
@@ -18,11 +18,17 @@ func newRecordImportManager(client *Client) *recordImportManager {
 	}
 }
 
+type importWorkerChannels struct {
+	recordChan <-chan Record
```

Comment:
why don't we call these fields "records", "errs" and "status" since they'll be referred to like `chans.records` which is a bit cleaner than `chans.recordChan`

Reply:
That makes sense. Updating.